lib/cc/engine/violation_decorator.rb

```diff
@@ -0,0 +1,41 @@
+module CC
+  module Engine
+    class ViolationDecorator < SimpleDelegator
+      MULTIPLIER_REGEX = %r{\[([\d\.]+)\/([\d\.]+)\]}
+
+      DEFAULT_REMEDIATION_POINTS = 200_000
+
+      def remediation_points
+        points = cop_list.fetch(cop_name, DEFAULT_REMEDIATION_POINTS)
+        if multiplier?
+          points * multiply_by
+        else
+          points
+        end
+      end
+
+      private
+
+      def multiply_by
+        result = message.scan(MULTIPLIER_REGEX)
+        result[0][0].to_i
```

Comment:
Sorry - I wasn't clear in my comment. I mean is there a way to get the score and threshold from the violation object?